src/sst/core/statapi/statlghistogram.h

```diff
@@ -0,0 +1,382 @@
+// Copyright 2009-2025 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2025, NTESS
+// All rights reserved.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+#ifndef SST_CORE_STATAPI_STATLGHISTOGRAM_H
+#define SST_CORE_STATAPI_STATLGHISTOGRAM_H
+
+#include "sst/core/sst_types.h"
+#include "sst/core/statapi/statbase.h"
+#include "sst/core/statapi/statoutput.h"
+#include "sst/core/warnmacros.h"
+
+namespace SST::Statistics {
+
+// NOTE: When calling base class members in classes derived from
+//       a templated base class.  The user must use "this->" in
+//       order to call base class members (to avoid a compiler
+//       error) because they are "nondependant named" and the
+//       templated base class is a "dependant named".  The
+//       compiler will not look in dependant named base classes
+//       when looking up independent names.
+// See: http://www.parashift.com/c++-faq-lite/nondependent-name-lookup-members.html
+
+/**
+    \class HistogramStatistic
+    Holder of data grouped into pre-determined width bins.
+    \tparam BinDataType is the type of the data held in each bin (i.e. what data type described the width of the bin)
+*/
+#define CountType   uint64_t
+#define NumBinsType uint32_t
```

Comment:
Unless macros are really needed, I would suggest `using`:
```
using CountType   = uint64_t;
using NumBinsType = uint32_t;

Reply:
Type aliasing is also my preference. The class implementation was largely lifted from the existing Histogram class - wasn't sure if modernizing to `using` keyword over `#define` macros was preferable. Will make those changes tomorrow.